apps/obs-web/src/views/AttributeSearchComp.vue

```diff
@@ -97,27 +98,30 @@ export default createComponent({
     const getSearchOptions = async () => {
         loading.value = true;
         sourceSearchOptions.value.length = 0;
-        const searchOptionResults = await masterDB.view(
-            'obs_web',
-            'wcgop_trips_compound_fields',
-            {reduce: true, include_docs: false, start_key: [selectedSearchType.value], end_key: [selectedSearchType.value, {}], group_level: 2} as any
-        );
-        sourceSearchOptions.value.push.apply(sourceSearchOptions.value, [...new Set(searchOptionResults.rows.map( (row: any) => {
-            if (row.key[1] === null || row.key[1] === undefined) {
-                return 'null';
-            } else if (['departureDate', 'returnDate'].includes(selectedSearchType.value)) {
-                return moment(row.key[1]).minute(0).hour(0).second(0).format();
+        if (!['departureDate', 'returnDate'].includes(selectedSearchType.value)) {
```

Comment:
on interesting, just curious so does the search page work with the errors page then? can't quite tell

Reply:
the search page loads trips to the Data / Trips tab, and errors will display for all trips selected on the trips tab.